validation_pipeline.ipynb

```diff
@@ -7,7 +7,7 @@
         "colab_type": "text"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    ISOSCAPE_OXYGEN_MEANS_FILENAME = "test_isoscape.tiff" #@param
</pre><p>should this be "test_isoscape"?  the old name seems very verbose, but this sounds like a temporary change.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tnc-br/ddf-isoscapes/pull/187/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
I don't think it should be anything, really. It seems to change with every PR.  Though, I kinda want to leave it as "test_isoscape.tiff" to avoid stamping and uploading the previous file.